clicmd

```diff
@@ -47,7 +51,7 @@ function role_to_group {
 
 # Lookup for role (privilege in BMC terms) by group name
 function group_to_role {
-  case "$1" in
+  case "${1:-}" in
```

Comment:
Empty `${1}` still gives error about unbound variable at line 59.
You should write an error message without `${1}` or use `${1:-}` there too.